istioctl/pkg/install/verify.go

```diff
@@ -453,3 +461,16 @@ func allOperatorsInCluster(client dynamic.Interface) ([]*v1alpha1.IstioOperator,
 	}
 	return retval, nil
 }
+
+// cleanupForGogo prepares an unstructed for github.com/gogo/protobuf's Unmarshal()
+func cleanupForGogo(un *unstructured.Unstructured) {
+	un.SetCreationTimestamp(meta_v1.Time{})             // operator_istio.UnmarshalIstioOperator chokes on these
+	un.SetManagedFields([]meta_v1.ManagedFieldsEntry{}) // operator_istio.UnmarshalIstioOperator chocks on the time in these
+}
+
+// isDeploymentName tells if this is a Deployment name created by "istioctl install" for the control plane
+func isDeploymentName(name string) bool {
+	return strings.HasPrefix(name, "istio-") ||
+		name == "istiod" || // 1.7.0 beta 1 uses this for master Istiod
```

Comment:
i feel uneasy about these sorts of tests being buried deep in the code. is it possible to make this more visible by declaring a const somewhere prominent? i also wonder if it's feasible to write a test to catch if something in the naming changes.